packages/firestore/src/platform/format_json.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as node from '../platform_node/node_format_json';
+import * as browser from '../platform_browser/browser_format_json';
+import { isNode } from '@firebase/util';
+
+// This file provides additional platform specific APIs. It acts an add on to
```

Comment:
missing "as" after acts

Reply:
Moved this to src/platform/README.md, which contains different text.